testcases/MachineConfig.py

```diff
@@ -325,6 +325,18 @@ def LparSetup(self):
             ioslot_drc_names = ",".join(ioslot_drc_names[:ioslot_drc_names.index("=")].split(",")[:-1]) \
                                if "=" in ioslot_drc_names else ioslot_drc_names
             self.cv_HMC.add_ioslot(ioslot_drc_names)
+        
+        if "perf=1" in self.machine_config:
+            conf = OpTestConfiguration.conf
+            if self.cv_HMC.is_perfcollection_enabled():
+                log.info("System is already booted with perf collection profile enabled")
+            else:
+                self.cv_HMC.hmc_perfcollect_configure()
+                if self.cv_HMC.is_perfcollection_enabled:
```

Comment:
since hmc_perfcollect_configure() returns a value, why not add a check for the same and avoid calling is_percollection_enabled() twice?